perf_test/graph/KokkosGraph_color.cpp

```diff
@@ -433,12 +433,13 @@ int main (int argc, char ** argv){
   }
   std::cout << "Sizeof(idx):" << sizeof(idx) << " sizeof(size_type):" << sizeof(size_type) << std::endl;
 
+  const int num_threads = params.use_openmp; // Assumption is that use_openmp variable is provided as number of threads
+  const int device_id = 0;
+  Kokkos::initialize( Kokkos::InitArguments( num_threads, -1, device_id ) );
 
 #if defined( KOKKOS_HAVE_OPENMP )
 
   if (params.use_openmp) {
-
-    Kokkos::OpenMP::initialize( params.use_openmp );
     Kokkos::OpenMP::print_configuration(std::cout);
```

Comment:
All of this print_configuration stuff should be done via `Kokkos::print_configuration(std::cout)`